pkg/pipeline/controller.go

```diff
@@ -263,8 +264,13 @@ func (c *Controller) UpdateStream(ctx context.Context, req *livekit.UpdateStream
 
 	// add stream outputs first
 	for _, rawUrl := range req.AddOutputUrls {
+		outputType := types.OutputTypeRTMP
```

Comment:
A `switch` statement may work better here as well. We also need to validate that the added URLs use the same protocol as the ones that are already setup.

Reply:
same as above